HomeScreenLib/src/main/java/com/example/homescreenlib/ui/HomeActivity.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.homescreenlib.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import com.example.homescreenlib.R
+import com.example.homescreenlib.data.DashboardItems
+import com.example.homescreenlib.databinding.ActivityHomeBinding
+import com.example.homescreenlib.viewmodel.DashboardViewModel
+import com.example.homescreenlib.viewmodel.ViewModelFactory
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var viewModel: DashboardViewModel
```

Comment:
make viewModel of type val

Reply:
shifted viewModel to HomeFragment while using Navigation. 
used val in HomeFragment